CHANGELOG/CHANGELOG-1.10.md

```diff
@@ -26,3 +26,4 @@ When cutting a new release, update the `unreleased` heading to the tag being gen
 * [BUGFIX] [#1061](https://github.com/k8ssandra/k8ssandra-operator/issues/1061) Point to cass-config-builder 1.0.7 for arm64 compatibility
 * [ENHANCEMENT] [#956](https://github.com/k8ssandra/k8ssandra-operator/issues/956) Enable linting in the project
 * [BUGFIX] [#1102](https://github.com/k8ssandra/k8ssandra-operator/issues/1102) Update gRPC maximum receive size to 512MB. Note, the operator might need more max memory than the default to take advantage of this.
+* [BUGFIX] [#1109](https://github.com/k8ssandra/k8ssandra-operator/issues/1109) Correctly configure the cr.k8ssandra.io registry
```

Comment:
**Issue:** This should go before line 18 in the "unreleased" section.

Reply:
ops, I just realized how the file is structured.